src/c++/uds/src/uds/numeric.h

```diff
@@ -6,7 +6,24 @@
 #ifndef UDS_NUMERIC_H
 #define UDS_NUMERIC_H
 
+#ifndef VDO_UPSTREAM
+#include <linux/version.h>
+#undef VDO_USE_NEXT
+#if defined(RHEL_RELEASE_CODE) && defined(RHEL_MINOR) && (RHEL_MINOR < 50)
+#if (RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(9, 0))
+#define VDO_USE_NEXT
+#endif
+#else /* !RHEL_RELEASE_CODE */
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 11, 0))
+#define VDO_USE_NEXT
+#endif
+#endif /* !RHEL_RELEASE_CODE */
+#endif /* !VDO_UPSTREAM */
+#ifdef VDO_USE_NEXT
+#include <linux/unaligned.h>
+#else
 #include <asm/unaligned.h>
+#endif
```

Comment:
I would like to alphabetize this properly, with linux/unaligned.h  appearing below linux/types.h. 
However we should probably fix that up separately because the upstream patch was script-generated and replaced the line without moving it. (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=5f60d5f6bbc12e782fac78110b0ee62698f3b576)

For clarity, this is a comment that requires no code change.